src/gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import plotly.graph_objects as go
+from sympy import Symbol, pi, E, sqrt, integrate, oo
+from abc import ABC, abstractmethod
+from scipy import stats
+import math
+
+
+class ProbabilisticDistribution(ABC):
+
+    @abstractmethod
+    def __init__(self):
+        pass
+
+    @abstractmethod
+    def save_figure(self):
+        pass
+
+
+class Visualizer:
+
+    @staticmethod
+    def visualize(x, y, title):
+        fig = go.Figure()
+        fig.add_trace(go.Scatter(x=x, y=y, name=title))
+        fig.update_xaxes(title="x")
+        fig.update_yaxes(title="y")
+        fig.update_layout(title=title)
+        fig.show()
+
+
+class GaussianProbabilisticSetting(ProbabilisticDistribution):
+
+    def __init__(self):
+        pro_x = np.linspace(-5, 5, 100)
+        pro_y = []
+        for i in range(len(pro_x)):
+            pro_y.append(float(Distribution.gauss_np(pro_x[i])))
+        pro_y = np.array(pro_y)
+
+        self.prob = {"x": pro_x, "y": pro_y}
+
+        x = Symbol('x')
+        t = Symbol('t')
+
+        # integrate gaussian pdf
+        F = integrate(Distribution.gauss_sp(t), (t, -oo, x))  # gaussian cumulative density function (gaussian cdf)
+
+        dis_y = []
+        t = np.linspace(-5, 5, 100)
+        for i in range(len(t)):
+            dis_y.append(float(F.subs({x: t[i]})))
+        dis_x = pro_x
+        dis_y = np.array(dis_y)
+        self.dist = {"x": dis_x, "y": dis_y}
+
+    def save_figure(self):
```

Comment:
Actually, it doesn't save and it shows figure.  Do following instructions referring cv_dnn

- define both save_figure and show_figure. 
- define PROJECT_ROOT (/workplace), FIGS_ROOT (/workplace/figs) variables in /workplace/config/const.py 
- import FIGS_ROOT in /workplace/config/\_\_init\_\_.py 
- import FIGS_ROOT in this file like: 
```
from config import FIGS_ROOT
```
- mkdir FIGS_ROOT to save figure (use makedirs(exist_ok=True) function) in main function

Reply:
Write args, returns in docstring. Also, write return type like:
```
def save_figure(self) -> (return_type):
```